service/patient-reminder.service.js

```diff
@@ -556,3 +609,29 @@ function transformZeroVl(vl) {
     return vl;
   }
 }
+
+function getRestResource(path) {
+  var protocol = config.openmrs.https ? 'https' : 'http';
+  var link =
+    protocol + '://' + config.openmrs.host + ':' + config.openmrs.port + path;
+  return link;
+}
+
+function getAllEncounters(patient_uuid) {
+  var uri = getRestResource(
```

Comment:
Move the encounters request resource to encounters service in openmrs-rest folder. That way it can be resusable